src/react/components/pages/modelCompose/modelCompose.tsx

```diff
@@ -252,89 +253,95 @@ export default class ModelComposePage extends React.Component<IModelComposePageP
+                                            setKey="multiple"
+                                            selectionMode={SelectionMode.multiple}
+                                            layoutMode={DetailsListLayoutMode.justified}
+                                            isHeaderVisible={true}
+                                            selection={this.selection}
+                                            selectionPreservedOnEmptyClick={true}
+                                            onRenderDetailsHeader={onRenderDetailsHeader}
+                                            onRenderRow={this.onRenderRow}>
+                                        </DetailsList>
+                                        {this.state.nextLink && this.state.nextLink !== "*" && !this.state.hasText &&
+                                            <div className="next-page-container">
+                                                {
+                                                    this.state.isLoading ?
+                                                    <Spinner
+                                                        label={strings.modelCompose.loading}
+                                                        className="commandbar-spinner"
+                                                        labelPosition="right"
+                                                        theme={getDefaultDarkTheme()}
+                                                        size={SpinnerSize.small}>
+                                                    </Spinner>
+                                                    :
+                                                    <PrimaryButton
+                                                        className="next-page-button"
+                                                        onClick={this.getNextPage}>
+                                                        <FontIcon iconName="Down" style={{padding: "5px"}}>
+                                                        </FontIcon>
+                                                        <span>Load next page</span>
+                                                    </PrimaryButton>
+                                                }
+                                            </div>}
+                                    </div>
+                                    }
+                                </ViewSelection>
+                            </ScrollablePane>
+                        </div>
+                        <ComposeModelView
+                                ref={this.composeModalRef}
+                                onComposeConfirm={this.onComposeConfirm}
+                                />
+                    </Customizer>
+                </Fabric>
+                <PreventLeaving
+                    when={this.state.isComposing}
+                    message={"A composing operation is currently in progress, are you sure you want to leave?"}
```

Comment:
should be in strings (EN and ES)

Reply:
currently, all prevent leaving strings aren't in string localization. I'll make another PR for that